README.md

```diff
@@ -107,6 +107,23 @@ See the [getting started documentation](https://developer.amazon.com/docs/ask-to
 
 and many more ...
 
+----
+
+## FAQs
+
+- **How do I start Alexa Skills Kit Toolkit over SSH remote development?**
+    - Install [Remote - SSH](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh) extension on Visual Studio Code, then connect to remote machine over SSH.
```

Comment:
If they are using the remote development option from [VSCode docs](https://code.visualstudio.com/docs/remote/ssh), they should have had the `ssh` extension already installed. So maybe a `note/assumption` would do for this?